dashboard/modules/serve/sdk.py

```diff
@@ -11,10 +11,10 @@
 from ray.dashboard.modules.dashboard_sdk import SubmissionClient
 
 
-DEPLOY_PATH = "/api/serve/deployments/"
```

Comment:
for my information, why do we need to have this change?

Reply:
Just trying to keep it consistent. Some of the paths have a trailing slash, some don't, and if you send a request without a trailing slash it will fail which sometimes made it hard to debug. I thought it'd be better for all paths to have no trailing slash, lmk if it's better practice to have them all have a trailing slash.

Reply:
Actually on second thought this can break user code. I'll leave it for now.